src/Symfony/Resources/views/fieldtype/fielddefinition_edit.html.twig

```diff
@@ -13,6 +13,18 @@
         {{- form_widget(form.ReturnedType) -}}
     </div>
 
+    <div class="query-enable-pagination{% if group_class is not empty %} {{ group_class }}{% endif %}">
+        {{- form_label(form.EnablePagination) -}}
+        {{- form_errors(form.EnablePagination) -}}
+        {{- form_widget(form.EnablePagination) -}}
+    </div>
+
+    <div class="query-items-per-page{% if group_class is not empty %} {{ group_class }}{% endif %}">
+        {{- form_label(form.ItemsPerPage) -}}
+        {{- form_errors(form.ItemsPerPage) -}}
+        {{- form_widget(form.ItemsPerPage) -}}
+    </div>
```

Comment:
This code block seems to be redundant. See https://github.com/ezsystems/ezplatform-admin-ui/pull/1149 for more details.